pkg/strategy/controller.go

```diff
@@ -161,7 +161,7 @@ func enableArchiveButton(uiOpts map[string]interface{}, sOpts map[string]string)
 
 func disableDependenciesTab(uiOpts map[string]interface{}, storage string, depsEnabled *bool) {
 	// dependency tab is by default enabled and memory storage support it
-	if strings.EqualFold(storage, "memory") || (depsEnabled != nil && *depsEnabled == false) {
+	if strings.EqualFold(storage, "memory") || (depsEnabled != nil && *depsEnabled == true) {
```

Comment:
I think it needs to be `... || depsEnabled == nil || *depsEnabled == true` - as if the `depsEnabled` is nil, then it means dependencies are enabled by default, doesn't it?

Reply:
The normalization of deps spec runs before - so we assuming correct settings here